mappings/net/minecraft/GameAndWorldVersion.mapping

```diff
@@ -1,2 +1,4 @@
 CLASS net/minecraft/unmapped/C_xyxorzsk net/minecraft/GameAndWorldVersion
+	METHOD getSeriesId getSeriesName ()Ljava/lang/String;
```

Comment:
I think we shouldn't remap unobfuscated stuff
```suggestion
	METHOD getSeriesId getSeriesId ()Ljava/lang/String;
```

Reply:
has it been decided by the quilt mapping gods that unobfuscated things should keep their original name?

Reply:
Imo, if it's unobfuscated, it's probably for a reason (unless it's a `FunctionalInterface`)